.github/workflows/preview-cleanup.yml

```diff
@@ -0,0 +1,27 @@
+name: Doc Preview Cleanup
```

Comment:
Cool, I didn't know this!

Reply:
Do you need an SSH key (like `DOCUMENTER_KEY`) for this to work?

Reply:
doesn't seem like it just by skimming. I yoinked it from MCMCChains originally.